src/types/onyx/Account.ts

```diff
@@ -1,11 +1,39 @@
 import type {ValueOf} from 'type-fest';
 import type CONST from '@src/CONST';
+import type {TranslationPaths} from '@src/languages/types';
 import type DismissedReferralBanners from './DismissedReferralBanners';
 import type * as OnyxCommon from './OnyxCommon';
 
 /** Two factor authentication steps */
 type TwoFactorAuthStep = ValueOf<typeof CONST.TWO_FACTOR_AUTH_STEPS> | '';
 
+/** The role of the delegate */
+type DelegateRole = ValueOf<typeof CONST.DELEGATE_ROLE>;
+
+/** Model of delegate */
+type Delegate = {
+    /** The email of the delegate */
+    email: string;
+
+    /** The role of the delegate */
+    role: DelegateRole;
+
+    /** Authentication failure errors */
+    error?: TranslationPaths;
```

Comment:
Now that it's working, I need to figure out how to replicate - but when I got an error on a delegate in the delegate list, I was unable to dismiss that error. We should presumably let that go away.

Reply:
1. if you close the popover the error will go 
2. if you click the try again to switch account, the error will go away. Optimistic data clears the error

Reply:
i think it is good enough 👍

Reply:
you might have experienced this before i added the logic to clear errors